roamance/src/components/profile/profile-header.tsx

```
@@ -230,157 +237,246 @@ export function ProfileHeader({
                 </Avatar>
                 <div
-                  className={`absolute inset-0 bg-black/30 rounded-full transition-all duration-300 flex items-center justify-center backdrop-blur-sm ${isUploading ? 'opacity-100' : 'opacity-0 group-hover:opacity-100'} ${isUploading ? 'cursor-wait' : 'cursor-pointer'}`}
+                  className={`
+                    absolute inset-0 bg-black/40 rounded-full
+                    transition-all duration-300
+                    flex items-center justify-center backdrop-blur-sm
+                    ${isUploading ? 'opacity-100' : 'opacity-0 group-hover:opacity-100'}
+                    ${isUploading ? 'cursor-wait' : 'cursor-pointer'}
+                  `}
                   onClick={() => !isUploading && setIsUploadDialogOpen(true)}
                 >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)